test/ip_address_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Fields.IpAddressTest do
+  use ExUnit.Case
+  alias Fields.{IpAddressPlaintext, IpAddressHash, IpAddressEncrypted}
+
+  describe "types" do
+    test "IpAddressPlaintext.type is :string" do
+      assert IpAddressPlaintext.type() == :string
+    end
+
+    test "IpAddressEncrypted.type is :binary" do
+      assert IpAddressEncrypted.type() == :binary
+    end
+
+    test "IpAddressHash.type is :binary" do
+      assert IpAddressHash.type() == :binary
+    end
+  end
+
+  describe "cast" do
+    test "IpAddressPlaintext.cast accepts a string" do
+      assert {:ok, "168.212.226.204"} == IpAddressPlaintext.cast("168.212.226.204")
```

Comment:
`168.212.226.204` appears to be a reasonably popular `IPv4` Address in books:
https://www.google.com/search?q=168.212.226.204
![image](https://user-images.githubusercontent.com/194400/82748942-c17f6780-9d9d-11ea-8fc3-fedb963b048d.png)

Just checking that it wasn't some random web service. 🔍